scapy/layers/tls/handshake.py

```diff
@@ -716,15 +715,15 @@ def post_dissection(self, pkt):
         """
         s = self.tls_session
         if s.prcs and s.prcs.key_exchange.no_ske:
-            logging.info("USELESS SERVER KEY EXCHANGE")
+            log_runtime.info("USELESS SERVER KEY EXCHANGE")
```

Comment:
Could you avoid ALL CAPS MESSAGES? ;-)

Reply:
Also, I'm not sure if it would be a good idea, but that's a suggestion: maybe it could help to add a bit of context here: if the underlayer contains IP or IPv6 and  TCP layers, add something like:

```Python
pkt_info = self.firstlayer().sprintf("{IP:%IP.src%}{IPv6:%IPv6.src%}"
                                     "{TCP::%TCP.sport%}{UDP::%UDP.sport%} > "
                                     "{IP:%IP.dst%}{IPv6:%IPv6.dst%}"
                                     "{TCP::%TCP.dport%}{UDP::%UDP.dport%}")
log_runtime.info("Useless server key exchange [%s]", pkt_info)
```

Maybe `pkt_info` can be replaced by `pkt.firstlayer().summary()`.

Reply:
I committed the changes, though I preferred not to weigh down the logs with additional information from outside the TLS layer.